jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/AuthParameters.java

```diff
@@ -24,7 +24,7 @@
  * A bean that configures authorization credentials to be used for a registry. This is configurable
  * with Groovy closures and can be validated when used as a task input.
  */
-public class AuthConfiguration {
+public class AuthParameters {
```

Comment:
just a question: what is the reasoning for this rename?

Reply:
Consistency (`ContainerParameters`, `ImageParameters`, `AuthConfiguration` -> `AuthParameters`)